### PR TITLE
Dev epam

### PR DESCRIPTION
**Pull request by IvanSakharovskii**

### CODE REVIEW

File: js/controller.js [hunk 1]
```diff
@@ -331,6 +331,12 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       }
     },
 
+    /**
+     * @description
+     * Should be used with deviceorientation event listener.
+     * Uses for video 360 on mobile devices for setting necessary coordinates (relevant with start device orientation)
+     * @param e - event
```
Comment (pilievOoyala) [inline js/controller.js]: Can we change this to the format we use in other JSDoc.  @param {object} e The event object.

Comment (KseniiaSokolovaEpam) [reply js/controller.js]: \+

**Comment by pilievOoyala** [comment]
LGTM

**Comment by aldigun** [comment]
LGTM.

**Pull request merged**